display_applications/ucsc/trackhub.xml

```diff
@@ -0,0 +1,6 @@
+<display id="ucsc_trackhub" version="1.0.0" name="display at Track Hub UCSC">
+    <link id="main" name="main">
+        <url>https://genome.ucsc.edu/cgi-bin/hgHubConnect?hubUrl=${qp($hub_file.url + '/myHub/hub.txt')}&amp;hgHub_do_firstDb=on&amp;hgHub_do_redirect=on&amp;hgHubConnect.remakeTrackHub=on</url>
+        <param type="data" name="hub_file" url="galaxy_${DATASET_HASH}" allow_extra_files_access="True" strip_https="True"/>
```

Comment:
I just followed [the wiki](https://wiki.galaxyproject.org/Admin/Tools/External%20Display%20Applications%20Tutorial) about this.

What is advised for this / what are the cases?